tests/Utils/UtilsTest.php

```diff
@@ -399,4 +399,12 @@ public function testCategoryPathIsProperlyBuiltWhenMainCategoryIsInADeeperPath()
         $categoryPath = Utils::buildCategoryPath($breadCrumbs, $rootCategoryMock);
         $this->assertSame($expectedCategoryPath, $categoryPath);
     }
+
+    public static function callMethod($obj, $name, array $args)
+    {
+        $class = new \ReflectionClass($obj);
+        $method = $class->getMethod($name);
+        $method->setAccessible(true);
+        return $method->invokeArgs($obj, $args);
+    }
```

Comment:
This file is not a utils class for the tests, it's tests for the existing `Utils` class. This will need to be placed in a separate helper file.